demos/python_demos/common/models/model.py

```diff
@@ -0,0 +1,29 @@
+from time import perf_counter
+
+
+class Model:
+    def __init__(self, ie, model_path, logger=None, batch_size=None):
```

Comment:
So, https://github.com/openvinotoolkit/open_model_zoo/pull/1639 needs `batch_size` argument as well?

Reply:
As we discussed this before, good practice is to use batch=1 and async pipeline. So , do we really need batch size here ?

Reply:
Default value is set for `1`. Anyone disagree?

Reply:
Fine with me. My question was - do we really need this parameter, but I'm fine with leaving it and set to 1 by default

Reply:
I think it would be better to make it configurable than force it to 1 or any other value